bitswap-tuning/test/common.go

```diff
@@ -0,0 +1,173 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ipfs/go-cid"
+	ipld "github.com/ipfs/go-ipld-format"
+
+	"github.com/testground/sdk-go/runtime"
+	"github.com/testground/sdk-go/sync"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+
+	"github.com/ipfs/test-plans/bitswap-tuning/utils"
+)
+
+func parseType(ctx context.Context, runenv *runtime.RunEnv, client *sync.DefaultClient, h host.Host, seq int64) (int64, utils.NodeType, int, error) {
```

Comment:
FYI some of this `parseType` function and `getNodeSetSeq` is most probably redundant, as with the latest SDKs we have 2 types of global sequence:
1. one unique global sequence for every single test plan instance (no matter which `group` it comes from)
2. one unique group sequence for every instance within a group (something that this part of the code appears to be implementing, as we didn't have it before).

See https://github.com/testground/sdk-go/blob/master/run/init_context.go#L36-L37